py3status/modules/speedtest.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+"""
+Do a bandwidth test with speedtest-cli.
+
+Configuration parameters:
+    button_share: button to open the result link (default None)
```

Comment:
(Conflicts) I'm trying to figure out if it's better to keep this `button_share` config or to use undocumented `{share}` placeholder instead + `index` so when users click `share`, it would open up an url... The latter requires users to touch `format`. I want to see if we can default this without `button_share` config too.